internal/cephfs/clone.go

```diff
@@ -84,19 +90,19 @@ func createCloneFromSubvolume(ctx context.Context, volID, cloneID volumeID, volO
 		util.ErrorLog(ctx, "failed to clone snapshot %s %s to %s %v", volID, snapshotID, cloneID, cloneErr)
 		return cloneErr
 	}
-	var clone CloneStatus
-	clone, cloneErr = getCloneInfo(ctx, volOpt, cr, cloneID)
+
+	cloneState, cloneErr := volOpt.getCloneInfo(ctx, cr, cloneID)
```

Comment:
why not go with the current implementation of returning the struct, currently we are acting only on the clone status maybe in the future we might make use of other fields also.

Reply:
There is currently no need for the fields, so no reason to have them. We can always add them later if they are useful. I don't like exposing internals of the go-ceph CephFS Admin package to higher layers, and prefer to keep the internal API as slim as possible.